juju/juju
# 'update--public-cloud' can update client and controller copies.

## Description of change

Originally, 'update-public-clouds' operated on client only.
This PR allows to update public cloud on the controller reducing the 2-step process described in the bug  down to 1-step.

Remove command alias 'update-clouds' as it is too close to a completely different 'update-cloud' command and is causing confusion.

As a drive-by, renamed some test variables that were conflicting with import names. 

## QA steps

1. bootstrap controller on a public cloud
2. 'juju update-public-clouds -c mycontroller'  says controller clouds are up to date
3. Ensure that there is a difference between your controller public cloud (I had to do db surgery, removing one region, for example)
4.  'juju update-public-clouds -c mycontroller' updates controller cloud definition (by adding that region)

## Documentation changes

as above

## Bug reference

https://bugs.launchpad.net/juju/+bug/1844440
